cmd/new_test.go

```diff
@@ -35,6 +36,50 @@ func newMockTime() time.Time {
 	return time.Date(2021, 5, 22, 13, 30, 10, 5, time.UTC)
 }
 
+func TestEnvOnNew(t *testing.T) {
+	cfg := newTestConfig()
+	cfg.Post = []core.PostProcessConfig{
+		{Key: "author", Value: "{{.Env.AUTHOR}}"},
```

Comment:
Just to make sure here, can you use a different key other then author? I just feel if I read this again in 6 months I might think it was just reusing the env var key which is not true.